src/components/Modal.js

```diff
@@ -140,7 +140,7 @@ export default class Modal extends Component {
         this.portal.closeWithTimeout();
       }
 
-      setTimeout(() => this.removePortal, closesAt - now);
+      setTimeout(() => this.removePortal(), closesAt - now);
```

Comment:
That is correct. You can remove the arrow function and it should work.

`setTimeout(this.removePortal, closesAt - now);`